proposals/params-collections.md

```diff
@@ -55,11 +55,12 @@ The *type* of a parameter collection shall be one of the following valid target
 - A *type* with an appropriate *[create method](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-12.0/collection-expressions.md#create-methods)*,
   which is at least as accessible as the declaring member, and with a corresponding *element type* resulting from that determination
 - A *struct* or *class type* that implements `System.Collections.IEnumerable` where:
-  - The *type* has a constructor that can be invoked with no arguments, and the constructor is at least as accessible as the declaring member, and
-  - The *type* has an instance (not an extension) method `Add` that can be invoked with a single argument of
-    the [*iteration type*](https://github.com/dotnet/csharpstandard/blob/draft-v9/standard/statements.md#1395-the-foreach-statement),
-    and the method is at least as accessible as the declaring member,
-  in which case the *element type* is the *iteration type*
+  - The *type* has a constructor that can be invoked with no arguments, and the constructor is at least as accessible as the declaring member.
+  - The *type* has an instance (not an extension) method `Add` where:
+    - The method can be invoked with a single argument and the corresponding parameter is either by value or `in`.
```

Comment:
>and the corresponding parameter is either by value or `in`. [](http://example.com/codeflow?start=55&length=59)

I am not sure where ref-ness restriction came from. Why is it needed?

Reply:
Probably the bullet point should say the following instead:
- The method can be invoked with a single value (vs. ref/out/in) argument.